### PR TITLE
Feature/erssup 69282

### PR DESCRIPTION
**Pull request by Nayira-Sanchez**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/e-referrals-service-api.yaml [hunk 1]
```diff
@@ -118,15 +118,15 @@ info:
...
+    For more information, please see [Network access](#api-description--network-access).
 
     ### Healthcare worker, user-restricted access
 
     This access mode is [in production, beta](https://digital.nhs.uk/developer/guides-and-documentation/reference-guide#statuses). We are onboarding Partners to use it.
     
     This access mode allows you to use [all endpoints](#api-endpoints) . A human user must be present and authenticated.
     
-    For more information, please see [Network access](#api-description__network-access).
+    For more information, please see [Network access](#api-description--network-access).
```
Comment (nhsd-david-wass) [inline specification/e-referrals-service-api.yaml:130]: A bit picky as there are only 2 uses of it, but its probably better to move this to populate_paceholders and create [[NYPERLINK_NETWORK_ACCESS]]

File: specification/components/stu3/schemas/headers/request/BusinessFunctionOBO.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    - 'REFERRING_CLINICIAN_ADMIN'
+    - 'REFERRING_ADMIN'
+    - 'COMMISSIONER'
+    - 'SERVICE_PROVIDER_CLINICIAN_ADMIN'
+    - 'SERVICE_PROVIDER_CLINICIAN'
+    - 'SERVICE_PROVIDER_ADMIN'
+    - 'INFORMATION_ANALYST'
+    - 'BOOKING_MANAGER'
+    - 'ADDITIONAL_REQUIREMENTS_MANAGER'
+  example: 'SERVICE_PROVIDER_CLINICIAN_ADMIN'
```
Comment (nhsd-david-wass) [inline specification/components/stu3/schemas/headers/request/BusinessFunctionOBO.yaml:19]: You'll need this file in r4/schemas/headers/requests as well - it's the reason your build is failing !

**Review by nhsd-david-wass — CHANGES_REQUESTED** [review]
Just a couple of comments

File: specification/e-referrals-service-api.yaml (hunk 1 above)
Comment (Nayira-Sanchez) [reply specification/e-referrals-service-api.yaml:130]: Done

**Review by nhsd-david-wass — DISMISSED** [review]

**Review by nhsd-jack-wainwright — COMMENTED** [review]
Looks good to me 🙂

**Review by stuandgray — APPROVED** [review]

**Pull request merged**